Dockerfile

```diff
@@ -93,7 +93,8 @@ ENV _APP_SERVER=swoole \
     _APP_FUNCTIONS_TIMEOUT=900 \
     _APP_FUNCTIONS_CONTAINERS=10 \
     _APP_SETUP=self-hosted \
-    _APP_VERSION=$VERSION
+    _APP_VERSION=$VERSION \
+    _APP_MAINTENANCE_INTERVAL=10
```

Comment:
Let me know what time units are we using here, we need to update this option in the homepage docs.

Reply:
I believe if we use seconds, we can stay consistent with the epoch time conventions